runner_benchmark/har_taskset.py

```diff
@@ -14,24 +15,19 @@ class HarFileTaskSet(TaskSet, utils.QuestionnaireMixins):
     def __init__(self, parent):
         super().__init__(parent)
 
+        self.base_url = self.locust.client.base_url
+
         self.har_filepath = os.environ.get('HAR_FILEPATH', 'requests.har')
-        self.eq_id = None
-        self.form_type_id = None
+        self.eq_id = 'test'
```

Comment:
?

Reply:
Suggestions welcome on how to get the values now that are no longer in the url?

Reply:
Argument or ENV?